 <launch>
    <!-- joy node -->
        <node respawn="true" pkg="joy" type="joy_node" name="wr_joy">
        	<param name="dev" type="string" value="/dev/input/js0" />
        	<param name="deadzone" value="0.0" />
        	<param name="autorepeat_rate" value="40"/>
        </node>
        
    <!-- Axes -->
        <param name="threshold" value="-0.5" type="double"/>
        <param name="acc_limit" value="0.5" type="double"/>
        <param name="dec_default" value="0.25" type="double"/>
        <param name="vel_limit" value="0.5" type="double"/>
        <param name="vel_acc" value="2" type="int"/>
        <param name="vel_dec" value="3" type="int"/>
        <param name="vel_angular" value="0" type="int"/>
        <param name="scale_linear" value="0.25" type="double"/>
        <param name="scale_angular" value="0.45" type="double"/>  
        <param name="safe_distance" value= "0.1" type="double"/>      
        <node pkg="wrp_joy" type="wrp_joy_node" name="teleop"/>
 </launch>
